src/state_title_screen.c

```diff
@@ -450,15 +450,29 @@ void title_screen_draw_press_start(void) {
     gDPSetCombineMode(gMainGfxPos++, G_CC_MODULATEIA_PRIM, G_CC_MODULATEIA_PRIM);
     gDPSetPrimColor(gMainGfxPos++, 0, 0, 248, 240, 152, TitleScreen_PressStart_Alpha);
     gDPPipeSync(gMainGfxPos++);
-    gDPLoadTextureBlock(gMainGfxPos++, TitleScreen_ImgList_PressStart, G_IM_FMT_IA, G_IM_SIZ_8b, 128, VAR_1, 0, G_TX_NOMIRROR | G_TX_WRAP,
-              G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD);
+    gDPLoadTextureBlock(gMainGfxPos++, TitleScreen_ImgList_PressStart, G_IM_FMT_IA, G_IM_SIZ_8b, 128, VAR_1, 0,
+                        G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD,
+                        G_TX_NOLOD);
     gSPTextureRectangle(gMainGfxPos++, 384, 548, 896, VAR_2, G_TX_RENDERTILE, 0, 0, 0x0400, 0x0400);
     gDPPipeSync(gMainGfxPos++);
 }
 
 #if VERSION_IQUE
-INCLUDE_ASM(void, "state_title_screen", title_screen_draw_copyright);
+#define RECT_SIZE 0x28
+#define YL_BASE 724
+#define YH_BASE 764
+#define COPYRIGHT_TEX_CHUNKS 4
+#define COPYRIGHT_IMG &TitleScreen_ImgList_Copyright[k]
```

Comment:
Maybe at least make these a function-like macro, so it's more obvious it depends on `k` /`i`?
```suggestion
#define COPYRIGHT_IMG(k, i) &TitleScreen_ImgList_Copyright[k]
```